### PR TITLE
chore(dashboard): include dashboard on navbar; try fixing bugs

### PR DESCRIPTION
**Pull request by luabida**

### CODE REVIEW

File: src/templates/vis/dashboard.html [hunk 1]
```diff
@@ -366,6 +382,7 @@
                                 var opt = document.createElement('option');
                                 opt.value = geocode;   
                                 opt.innerHTML = `${result.uf} - ${result.municipio} (${geocode})`;
+                                opt.selected = false; // not working
```
[the remark this answers is not in the document]
Comment (fccoelho) [reply src/templates/vis/dashboard.html:385]: @luabida remember my suggestion from a while ago that we should abandon dropdowns and adopt a simple nested tree?

[the remark this answers is not in the document]
Comment (luabida) [reply src/templates/vis/dashboard.html:385]: Yes, it is on my TODOs

**Comment by fccoelho** [comment]
@luabida could you drop some screenshots here, when needed?  it will make it easier to see how the layout is evolving. and it's  easier than rendering the HTML in our brains :-D

**Comment by luabida** [comment]
@fccoelho I'm current working on the predictions filter on the JS side, I'm replacing the entire logic for the Prediction compatibilities so it become independent from the view. Mainly because JS doesn't work very well with the Django model object. I've also replaced the dropdowns by the checkbox as asked
image

**Comment by fccoelho** [comment]
It's much better, the new design! thanks!

**Comment by luabida** [comment]
this is being way harder than I expected, trying a different approach rn

**Comment by luabida** [comment]
I finally finished the logic for the dashboard, only some aesthetic aspects of the page were left to fix, tomorrow morning I'll be able of merging this PR

**Review by fccoelho — APPROVED** [review]

**Comment by luabida** [comment]
I'm implementing this PR in production and fix the upcomming issues in later PRs

**Pull request merged**